index.js

```diff
@@ -1,7 +1,7 @@
 import React, { useState, useContext, createContext } from "react";
 
-export default function createStore(store = {}, callbacks = {}) {
-  const keys = Object.keys(store);
+export default function createStore(defaultStore = {}) {
```

Comment:
I would put the option to generate callbacks also outside the Provider level. This way there are both options.

```suggestion
export default function createStore(defaultStore = {}, defaultCallbacks = {}) {
```